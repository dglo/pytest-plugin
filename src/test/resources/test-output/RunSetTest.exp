tsts=2
errs=0
fail=0
time=0.0010
xml=true
