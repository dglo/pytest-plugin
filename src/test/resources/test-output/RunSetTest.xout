<testsuite errors="0" failures="0" name="unittest.TestSuite" tests="2" time="0.001">
  <testcase classname="__main__.TestRunSet" name="testEmpty" time="0.0002"></testcase>
  <testcase classname="__main__.TestRunSet" name="testSet" time="0.0004"></testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
