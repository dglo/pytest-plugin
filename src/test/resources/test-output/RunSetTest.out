testEmpty (__main__.TestRunSet) ... ok
testSet (__main__.TestRunSet) ... ok

----------------------------------------------------------------------
Ran 2 tests in 0.001s

OK
